From ced9c46e2f516d0b906c6c73e1daba5e563e644d Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Wed, 3 Aug 2016 20:36:51 +0800
Subject: [PATCH] ARM: configs: enable AP6XXX WiFi in rockchip_linux_defconfig

This patch enable rkwifi and AP6XXX series wifi chips config

Change-Id: I663df14011ce8402566d62c9e4d9b9a11b274fbe
Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
---
 arch/arm/configs/rockchip_linux_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/rockchip_linux_defconfig b/arch/arm/configs/rockchip_linux_defconfig
index 7ddd2aa51e9b..f577fe309499 100644
--- a/arch/arm/configs/rockchip_linux_defconfig
+++ b/arch/arm/configs/rockchip_linux_defconfig
@@ -186,6 +186,8 @@ CONFIG_HOSTAP_FIRMWARE=y
 CONFIG_HOSTAP_FIRMWARE_NVRAM=y
 CONFIG_RT2X00=y
 CONFIG_RT2800USB=y
+CONFIG_WL_ROCKCHIP=y
+CONFIG_AP6XXX=y
 CONFIG_MWIFIEX=y
 CONFIG_MWIFIEX_SDIO=y
 # CONFIG_INPUT_MOUSEDEV is not set
-- 
2.35.3

